modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/scaling-a-user-provisioned-cluster-with-the-bare-metal-operator.adoc
+:_content-type: PROCEDURE
+[id="upi-provisioning-new-hosts-in-a-upi-cluster_{context}"]
+= Provisioning new hosts in a user-provisioned cluster by using the BMO
+
+You can use the Bare Metal Operator (BMO) to provision bare-metal hosts in a user-provisioned cluster by creating a `BareMetalHost` custom resource (CR). 
+
+[NOTE]
+====
+To provision bare-metal hosts to the cluster by using the BMO, you must set the `spec.externallyProvisioned` specification in the `BareMetalHost` custom resource to `false`.
+====
+
+.Prerequisites
+
+* You created a user-provisioned bare-metal cluster.
+* You have baseboard management controller (BMC) access to the hosts.
+* You deployed a provisioning service in the cluster by creating a `Provisioning` CR.
+
+.Procedure
+
+. Create the `Secret` CR and the `BareMetalHost` CR.
+
+.. Save the following YAML in the `bmh.yaml` file:
++
+[source,yaml]
+----
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: worker1-bmc
+  namespace: openshift-machine-api
+type: Opaque
+data:
+  username: <user_name>
+  password: <password>
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: worker1
+  namespace: openshift-machine-api
+spec:
+  bmc:
+    address: <protocol>://<bmc_url>
+    credentialsName: "worker1-bmc"
+  bootMACAddress: <nic1_mac_address>
+  customDeploy:
+    method: install_coreos
+  online: true <1>
+  userData:
+    name: worker-user-data-managed
+    namespace: openshift-machine-api
+
+----
+<1> You must set the `spec.externallyProvisioned` specification in the `BareMetalHost` custom resource to `false`.
```

Comment:
... or omit it?

You don't have this field in your sample (it defaults to false anyway).

Reply:
Oops! I will include it in the sample anyway. I can mention the default is false too.